### PR TITLE
Added GSoC blog posts for remaining weeks

### PR DESCRIPTION
**Pull request by antrikshmisri**

### CODE REVIEW

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  * Add Accordion2D UI element to the UI sub-module
+  * Add SpinBox2D UI element to the UI sub-module
+
+Objectives Completed
+--------------------
+
+
+-  **Add support for Horizontal Layout**
+
+   Added support for Horizontal Layout in the layout module. This layout allows the user to stack actors in a horizontal fashion. Primarilty, should be used for laying out UI elements as there is no meaning of horizontal/vertical in 3D space.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:46]: typo: `Primarily`

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+   *Pull Requests:*
+
+   -  **Horizontal Layout:** https://github.com/fury-gl/fury/pull/480
+   -  **Ribbon Representation demo:** https://github.com/fury-gl/fury/pull/480
+
+- **Add support for Vertical Layout**
+
+  Added support for Vertical Layout in the layout module. This layout allows the user to stack actors in a vertical fashion. Primarilty, should be used for laying out UI elements as there is no meaning of horizontal/vertical in 3D space.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:55]: typo: `Primarily`

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+  *Pull Requests:*
+
+  - **Vertical Layout:** https://github.com/fury-gl/fury/pull/479
+  - **Vertical Layout demo:** https://github.com/fury-gl/fury/pull/479
+
+- **Add support for Layout along X, Y, Z axes**
+
+  Added support for Layout along x, y, z axes. Allows user to layout different actors along any given axes. Also allows users to switch the stacking order by passing a axis+ or axis- to the constructor.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:64]: Also, it....

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+  *Pull Requests:*
+
+  - **X, Y, Z axes Layout:** https://github.com/fury-gl/fury/pull/486
+  - **X, Y, Z axes Layout demo:** https://github.com/fury-gl/fury/pull/486
+
+- **Add Tree2D UI element to the UI sub-module**
+
+  Added Tree2D UI element to the UI sub-module. This allows user to visualize some data in a hierarchical fashion. Each node inside the tree can have N child nodes and the depth can be infinite. Each node can be clicked to trigger a user defined callback to perform some action. Tests and two demos were added for this UI element. Below is a screenshot for reference:
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:73]: Where is the screenshot ?

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+        :width: 200
+        :height: 200
+ 
+   *Pull Requests:*
+
+   - **Add Card2D UI element to UI sub-module:**  https://github.com/fury-gl/fury/pull/398
+
+-  **Resize Panel2D with WindowResizeEvent or from corner placeholder**
+
+   Currently, the size of the Panel2D is static and cannot be changed dynamically. The size is passed in during the initialization and cannot be changed easily at runtime. This PR adds support for resizing the Panel2D dynamically by adding a placeholder icon at the bottom right corner of the panel. This icon canbe click and dragged on to change the size accordingly. Other than this, the panel also retains a specific size ratio when the window is resized. This means if the window is resized in any direction the panel adapts itself w.r.t the updated size. This is done by adding relevant observers for the WindowResizeEvent and binding the relevant callback to it. Below is a quick demonstration:
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:125]: typo: `can be`

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+    *Pull Requests:*
+
+    - **First Evaluation:** https://github.com/fury-gl/fury/pull/477
+
+    - **Second Evaluation:** https://github.com/fury-gl/fury/pull/494
+
+Timeline
+--------
+
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:167]: The table did not work, can you check why ? See below:
Capture d’écran 2021-09-11 à 15 24 47

File: docs/source/posts/2021/2021-08-16-week-11-antriksh.rst [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+.. post:: August 16 2021
+   :author: Antriksh Misri
+   :tags: google
+   :category: gsoc
+
+What did I do this week?
+------------------------
+Below are the tasks that I worked on:
+
+* `Created PR for sprite sheet animation <https://github.com/fury-gl/fury/pull/491>`_ : This Pr adds support for playing animations from a sprite sheet. This feature will be used in Card2D to create a tutorial in which the card will show the animation in the image box. Previously, the utility functions for this were added directly inside the tutorial but now they are refactored to go in their respective modules.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-16-week-11-antriksh.rst:13]: `PR` instead

File: docs/source/posts/2021/2021-08-16-week-11-antriksh.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+* `Finalized the x, y, z layouts <https://github.com/fury-gl/fury/pull/486>`_ : The Pr that adds these layouts needed some updates for it to work as intended. These changes were added and this Pr is ready to go.
```
Comment (skoudoro) [inline docs/source/posts/2021/2021-08-16-week-11-antriksh.rst:14]: `PR` instead

**Review by skoudoro — COMMENTED** [review]
Hi @antrikshmisri,

Overall, it looks good. Thank you for this. See below for some minor comments. could you also rebase. After those changes, it will be ready to go. Thanks!

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst (hunk 1 above)
Comment (antrikshmisri) [reply docs/source/posts/2021/2021-08-23-final-work-antriksh.rst]: fixed

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst (hunk 4 above)
Comment (antrikshmisri) [reply docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:73]: Added.

File: docs/source/posts/2021/2021-08-23-final-work-antriksh.rst (hunk 6 above)
Comment (antrikshmisri) [reply docs/source/posts/2021/2021-08-23-final-work-antriksh.rst:175]: The table had some formatting issues. It should be fine now.

**Review by skoudoro — APPROVED** [review]
Thank you for the update @antrikshmisri!

LGTM, merging.

**Pull request merged**